### DISCUSSION TITLE
Are versions below 1.10.2 actually supported?

### DISCUSSION THREAD

**Discussion by Southpaw1496** [gorilla-devs/ferium#104] [Q&A] [answered]
In the profile creator wizard, it asks, "what version of Minecraft do you play"? and the earliest version it offers is 1.10.2. Now, every so often I like to play with a mod that still hasn't been updated past 1.7.10 (they're working on it, but it's a big job), so I manually set the version to 1.7.10 with the `--game-version` flag. However, although some mods import successfully (such as the [Lord of the Rings mod](https://www.curseforge.com/minecraft/mc-mods/the-lord-of-the-rings-mod-legacy), some mods, especially those that have files for newer versions of Minecraft as well as the old, return the error `The project/repository is not compatible`. I thought that this might be because the CurseForge API wasn't returning the old files, but I examined the raw JSON returned by the API, and the files are clearly there. So is the reason for this error instead that versions below 1.10.2 aren't actually supported, or is there some other reason?

**Comment by theRookieCoder** [comment, marked as answer]
Before 1.13, basically all mod were for Forge. I think that's why a lot of old mods don't flag Forge as a compatible mod loader, because it was implied.

So now, Ferium can't tell what mod loader old mods support because they're not specified. There was an issue about this at #12 (wow that's old) and the solution is to disable the mod loader check for mods that have this problem

**Reply by JustSimplyKyle** [reply]
Maybe when the modloades aren't specified in 1.12.2 and bellow, ferium defaults to forge.

**Reply by JustSimplyKyle** [reply]
That would make the assumption that all rift and legacy fabric mods all specifically specify their modloader

**Reply by theRookieCoder** [reply]
Yeah I did consider doing that but it would probably just lead to more problems. Letting the user choose is simpler imo
